x/upgrade/abci_test.go

```diff
@@ -23,6 +25,7 @@ type TestSuite struct {
 	module  module.AppModule
 	ctx     sdk.Context
 	cms     store.CommitMultiStore
+	FlagUnsafeSkipUpgrade string
```

Comment:
Why not use a boolean here instead?

Reply:
We're not declaring the bool here,instead declaring the key and assigning a value. Coz we're reading the value of flag in BeginBlock,so we need to just assign the key and check it's being read in beginblock